codecov.yml

```diff
@@ -0,0 +1,25 @@
+coverage:
+  status:
+    project:
+      default:
```

Comment:
Hmm, isn't clear that its putting default into one bucket and these files into another, need to look into this...

Reply:
Looks like the exclusions must be manually declared, was able to see this by looking at the coverage configuration at the commit level - https://app.codecov.io/gh/opensearch-project/security/pull/3196/commits

Reply:
I've resolved this by seperating the plugin out with its own exclusions which is working as intented

![image](https://github.com/opensearch-project/security/assets/2754967/b4b45761-c83c-48d9-890a-50f4efd323c3)